src/vcModals.cpp

```diff
@@ -851,16 +851,20 @@ void vcModals_DrawBindings(vcState *pProgramState)
     vcHotkey::DisplayBindings(pProgramState);
 
     ImGui::Columns(3, "ButtonRow");
-    if (ImGui::Button(udTempStr("%s###bindingsClose" , vcString::Get("bindingsModalClose")), ImVec2(-1, 25)) || vcHotkey::IsPressed(vcB_Close))
-      ImGui::CloseCurrentPopup();
+    if (ImGui::Button(udTempStr("%s###bindingsRevert", vcString::Get("bindingsModalRevert")), ImVec2(-1, 25)) || vcHotkey::IsPressed(vcB_Cancel))
+      vcHotkey::StampBinds(true);
```

Comment:
Cancel implies the popup closes - otherwise it would be called Revert.

Reply:
That naming is to operate as much as possible as a catch-all for anything we'd map to the escape key, so closing popups or reverting changes

Reply:
My apologies, I misunderstood what was happening here.